### PR TITLE
Add semantic ui translated view

### PR DESCRIPTION
**Pull request by loic425**
This Pull request depends on this one:
https://github.com/whiteoctober/Pagerfanta/pull/216

### CODE REVIEW

File: View/SemanticUiTranslatedView.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'semantic_ui_translated';
+    }
+}
```
Comment (sampart) [inline View/SemanticUiTranslatedView.php]: It's a minor point, but please could you add a blank line at the end of the file, as this is something git prefers (http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file).

Comment (loic425) [inline View/SemanticUiTranslatedView.php]: @sampart 
Ok no problem :)

**Comment by sampart** [comment]
@loic425 Please could you take a look at why the tests are failing?  The failures are referencing (a mock of) your new view, so it looks related.

**Comment by loic425** [comment]
@sampart 
I think this is just the pargerfanta dependency on the new "SemanticUiView" class.
Do i need waiting for a new pagerfanta release ?

**Comment by sampart** [comment]
Thanks @loic425.  I've created a new Pagerfanta release, and tests are failing for different reasons now.  Over to you...

(Sorry not to have the time to help you debug your tests in any detail!)

**Comment by loicmobizel** [comment]
@sampart 
the tests are failing with twitter bootstrap views because of https://github.com/whiteoctober/Pagerfanta/commit/f30d50899eddcf6361b5aacfdcf8b866d14cb8d9

I think i'ts better to fix them in another PR. I can do it if you want to.

**Comment by loicmobizel** [comment]
@sampart 
On my branch, all tests are ok. 
Can we just relaunch build ?

**Comment by sampart** [comment]
@loic425 I've tried relaunching the build a couple of times now with no success unfortunately.  I will take a look at the Pagerfanta commit you mention when I get a minute, but I'm afraid things are pretty hectic at this end at the moment!  Sorry for the delays!

**Comment by loic425** [comment]
@sampart Do you think to have much time soon ?

**Comment by sampart** [comment]
Sorry, @loic425, it's not looking like I'll have time before January.  Sorry - things aren't normally quite this time-pressured here!  Feel free to put in a PR to update the tests if that's what needed (like I say, I've not had time to investigate myself) and I can merge that and this could then be rebased off the new work.

**Comment by loic425** [comment]
@sampart Do you have much time ?

**Comment by sampart** [comment]
Sorry about my lack of availability, @loic425!  Did you see my comment about you putting in an additional PR above?  (I'm afraid I'm still very short on time here :frowning_face: )

**Comment by sampart** [comment]
Hi @loic425, thanks for your patience!  I finally got some time to look at this.

At first, my tests were passing locally too - I deleted `composer.lock` and re-ran `composer install` and things started failing.  `composer.lock` isn't committed, so tests on Travis always run against the latest versions (subject to the version numbering in `composer.json`)

I created a branch which fixes the tests, pulled your branch, rebased it off mine and tests still passed, so I'm happy that you've not broken anything.

I'm therefore going to merge this.  Thanks again for contributing and apologies again for keeping you hanging for so long!!

**Comment by pjedrzejewski** [comment]
Thanks guys! This will be very useful for us, appreciate your work! :)

**Comment by loic425** [comment]
@sampart 
Great news ! Thank you for your help too.
@pjedrzejewski 
Sylius Resource and Grid bundles helps me every day, it's normal to contribute a little on sylius :)

**Comment by loic425** [comment]
@sampart 
When do you plan to make a new release ?

**Comment by sampart** [comment]
@loic425 Thanks for reminding me - I was just about to ask you if you needed one!  I've tagged release v1.0.8 now: https://github.com/whiteoctober/WhiteOctoberPagerfantaBundle/releases/tag/v1.0.8

Let me know if there's anything else you need.  Thanks

**Comment by loic425** [comment]
@sampart 
Thanks a lot ! It's ok for me
@pjedrzejewski @pamil 
I'm gonna do a pull request on sylius to use pagerfanta new release and remove the duplicated source from sylius :)

**Pull request merged**